src/lib/server.ml

```diff
@@ -86,7 +100,14 @@ module Authentication = struct
   let load meta_tokens =
     Deferred_list.while_sequential meta_tokens ~f:(function
       | `Path p -> load_file p
-      | `Inline (name, value) -> return [{name; value; comments = []}])
+      | `Inline (name, value) ->
+         let token = {name; value; comments = []} in
+         return begin match valid token with
+                       | true -> [token]
+                       | false ->
+                          Log.(s "Invalid character(s) in token: " % OCaml.string value @ warning);
+                          []
+                end)
```

Comment:
Please avoid the too long lines: https://github.com/hammerlab/style-guides/blob/master/ocaml.md#more-rules

Something like that looks better:

```ocaml
let token = {name; value; comments = []} in
let tokens =
  match valid token with
  | true -> [token]
  | false ->
    Log.(s "Invalid character(s) in token: " % OCaml.string value
         @ warning);
    [] in
return tokens
```